packages/webapp/__tests__/SquadFeedPage.tsx

```diff
@@ -231,15 +231,52 @@ Object.assign(navigator, {
 
 describe('squad header bar', () => {
   it('should show five of the squad member images', async () => {
-    const members = generateMembersResult().sourceMembers.edges;
+    Object.defineProperty(global, 'matchMedia', {
```

Comment:
this alone will make it render as laptop size?
Can't remember how we do different sizes 😅

Reply:
Yes